.github/workflows/plan-examples.py

```diff
@@ -10,14 +10,17 @@ def get_examples():
     """
     exclude = {
         'ai-ml/ray/terraform', # Skip until we fix CI script to test as per v5 convention
+        'ai-ml/ray/terraform/examples/pytorch', # Example relies on cluster created by parent blueprint
```

Comment:
these two utilize a data source to look up the EKS cluster that is created in the parent Ray example so they will never pass a plan on their own

Reply:
Can we add `workshop/*`  to exclude all the sub module and other workshop examples?

Reply:
ya, let me play with the regex - there are some places where there are modules local to the blueprint and even those local modules have sub-modules https://github.com/awslabs/data-on-eks/tree/main/analytics/terraform/datahub-on-eks/datahub-addon which will fail a plan as well

Reply:
ok should be fixed with https://github.com/awslabs/data-on-eks/pull/243/commits/b2fae25631fddd928ee18b5ef3cf2d2a1f9978f3 and would take effect after merge